src/edits.ts

```diff
@@ -178,6 +178,12 @@ async function addReleasesToTrack(appEditId: string, options: EditOptions, versi
     core.debug(`track=${options.track}`)
     if (options.userFraction) {
         core.debug(`userFraction=${options.userFraction}`)
+
+        // When all users are targeted (i.e. userFraction equal to 1) modify it to be undfined as required by the update
+        // function provided by androidPublisher.edit.tracks
+        if (options.userFraction === 1.0) {
+            options.userFraction = undefined;
```

Comment:
Please at least add a warning that userFraction is ignored. I don't think we want people getting into a habit of passing data that won't be used

Reply:
It should also only be a warning for "completed" releases. We still want to fail on invalid config (partial rollout at 100% is invalid)